package-lock.json

```diff
@@ -4,7 +4,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "akkaserverless-javascript-sdk",
```

Comment:
Not sure why `package-lock.json` has been updated, but it shouldn't be for this PR.